README.md

```diff
@@ -181,6 +181,13 @@ To add a resource or contribute, please check out our contribution guide [here](
 
 ---
 
+## C++ Algorithms
+
+- [Algorithms](https://cp-algorithms.com/index.html)
+- [GeeksforGeeks](https://www.geeksforgeeks.org/fundamentals-of-algorithms/)
```

Comment:
This one is algorithms in general rather than just C++. So, add a new topic Algorithm and place it under it

Reply:
You still need to resolve this @NoobSolver

Reply:
Check now, I have updated it

Reply:
I meant you need to add a new topic like C++ called Algorithms and under it add the resource for GeeksforGeeks.

So, rename Algorithms back to C++, add a new topic Algorithms and add it in the table of content, move the link for GeeksforGeeks to Algorithms, and rename the link from GeeksforGeeks to the page's title.

Let me know if you have questions

Reply:
I have updated it accordingly. Please review it,